kpi-tracker/resources/schema.sql

```diff
@@ -34,9 +34,16 @@ CREATE INDEX IF NOT EXISTS blocks_timestamp ON blocks (timestamp);
 -- All payday blocks (only exists for protocol version 4 and onwards).
 CREATE TABLE IF NOT EXISTS paydays (
   block INT8 PRIMARY KEY REFERENCES blocks(id) ON DELETE RESTRICT ON UPDATE RESTRICT,
-  total_stake INT8 NOT NULL,
+  total_equity_capital INT8 NOT NULL,
+  total_passively_delegated INT8 NOT NULL,
+  total_actively_delegated INT8 NOT NULL,
   num_bakers INT8 NOT NULL,
-  num_finalizers INT8 NOT NULL
+  num_open_bakers INT8 NOT NULL,
+  num_closed_bakers INT8 NOT NULL,
+  num_open_delegation_recipients INT8 NOT NULL,
+  num_closed_for_new_delegation_recipients INT8 NOT NULL,
+  num_finalizers INT8 NOT NULL,
+  num_delegators INT8 NOT NULL
```

Comment:
@abizjak just to fill you in: Lasse and I discussed this structure yesterday and it's sort of a compromise. The goal is to enable filtering in a stacked graph (bakers/delegators), (baker open, baker closed, open with delegators, closed with delegators), (baker stake, active delegator stake, passive delegator stake).

Of course there are alternative solutions that might provide more flexibility (record payday data per baker), but this solves the problem at hand and will perform well.